### PR TITLE
Add lockr v0.8.4 with npm autoupdate config added

### LINKED ISSUE #7528: [Request] Add lockr
**Library name:** tsironis/lockr
**Git repository url:** https://github.com/tsironis/lockr
**npm package url(optional):** 
**License(s):**The MIT License
**Official homepage:**https://github.com/tsironis/lockr
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
PR for #7528
@kennynaoh please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/tsironis/lockr
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/lockr
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 337
   - Count of watchers: 10
   - Count of forks: 26
 * NPM download stats (optional):
   - Downloads in the last day: 111
   - Downloads in the last week: 520
   - Downloads in the last month: 1731

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7528, cc @ThomasGroch

### CODE REVIEW

File: ajax/libs/lockr/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "author": {
+    "name": "Dimitris Tsironis",
+    "email": "tsironis.dimitris@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/tsironis/lockr.git"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/tsironis/lockr#readme",
```
Comment (PeterDaveHello) [inline ajax/libs/lockr/package.json]: @x09326 did you manually add this line? I didn't see this from its git repo or npm pkg, there is no need to add homepage link to its git repo.

Comment (x09326) [inline ajax/libs/lockr/package.json]: I downloaded it from npm, and I didn't change this line.

Comment (PeterDaveHello) [inline ajax/libs/lockr/package.json]: hmmmm ... I guess that newer npm added.

Comment (PeterDaveHello) [inline ajax/libs/lockr/package.json]: @x09326 you should try `pnpm` which will not touch the `package.json` from upstream.
See https://github.com/rstacruz/pnpm

cc @cdnjs/intern2

Comment (PeterDaveHello) [inline ajax/libs/lockr/package.json]: See the diff here:
package json

Comment (PeterDaveHello) [inline ajax/libs/lockr/package.json]: Let's remove this.

**Comment by x09326** [comment]
@PeterDaveHello I have already removed the ```homepage```. Please help me review it, thank you.

**Comment by PeterDaveHello** [comment]
Thanks.

**Pull request merged**